src/main/query/where.c

```diff
@@ -205,8 +273,13 @@ static int AerospikeQuery_Where_Add(AerospikeQuery *self, PyObject *py_ctx,
         free(val1_bytes);
     }
 
-CLEANUP_ON_ERROR1:
+CLEANUP_EXP_ON_ERROR:
 
+    if (exp_list) {
+        free(exp_list);
```

Comment:
Food for thought. There is use of both `cf_free` and `free` in the Python client. It would be good to standardize on one. The nice thing about using `cf_free` is you can easily instrument it should you need to.